app/src/main/java/com/github/polypoly/app/models/game/GameViewModel.kt

```diff
@@ -59,14 +137,33 @@ class GameViewModel(
                 // TODO: Remove this when the game is created from the lobby
                 GameRepository.game = Game.launchFromPendingGame(
                     GameLobby(
+                        rules = GameParameters(
+                            gameMode = GameRepository.game?.rules?.gameMode ?: GameMode.LAST_STANDING,
+                            maxRound = GameRepository.game?.rules?.maxRound,
+                        ),
                         admin = GlobalInstances.currentUser
                     )
                 )
                 GameRepository.player = Player(
                     GlobalInstances.currentUser,
-                    3000
+                    // hardcoded values for the testing
+                    3000,
+                    ownedLocations = mutableListOf(Game.gameInProgress?.getInGameLocation()?.get(0)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(1)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(2)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(5)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(8)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(9)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(10)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(11)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(12)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(13)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(14)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(15)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(16)!!,
+                        Game.gameInProgress?.getInGameLocation()?.get(17)!!,),
```

Comment:
It's not for ddebug, again when I created this PR all your PRs weren't merged and we just didn't have a working game, so it was IMPOSSIBLE for me to make my feature work any other way. If I removed that, you just couldn't test my feature AT ALL

Reply:
Ok I get why you left it in the pr. However as it remains specific to this feature and not to the entire game, I would still remove it from the code and for example add it in the description of your pr so that the people who review it can just copy paste it if they need it: it is still debug/test code so pushing it on main doesn't seem very right to do...